requirements.txt

```diff
@@ -2,5 +2,5 @@ numpy~=1.14
 scipy~=1.0
 scikit-learn~=0.19
 six~=1.10
-tensorflow~=1.4
```

Comment:
wouldn't this lead to the latest 1.x being installed already?

would `tensorflow~=1.12.1` or `tensorflow>=1.12.1,<=2.0.0` or something be better, to avoid potential incompatibilities?

Reply:
Yes, in practice we were using it with a newer version of `tensorflow` than this. The requirement change is to ensure that installations of `muffnn` avoid some security vulnerabilities in tensorflow.

I updated the requirements to require <2. In practice, I think we'll become incompatible before that, but I'm sure we would be incompatible in v2.